src/components/Abstractor/Abstractor.tsx

```diff
@@ -6,6 +6,8 @@ import Questionnaire from '../Questionnaire';
 import { QuestionnaireLoader } from '../../loaders/QuestionnaireLoader';
 import { LibraryLoader } from '../../loaders/libraryLoader';
 import executeElm from '../../utils/cql-executor';
+import { ValueSetLoader } from '../../loaders/ValueSetLoader';
+import resultsProcessing from '../../utils/results-processing';
```

Comment:
We can comment this out until you implement it to suppress the lint 

```suggestion
// import resultsProcessing from '../../utils/results-processing';
```